app/posts/find-teacher-training/2022-05-16-making-the-service-more-browseable.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Making the service more browseable
+description: An exploration into how we might use links instead of form-based questions
+date: 2022-05-16
+ogImage:
+  src: /find-teacher-training/making-the-service-more-browseable/todo.png
+  alt: TODO
+---
+
+![](flow-update.png "")
+
+We explored some alternative designs for the initial flow of the service, with the aim to make more of a browsing experience.
+
+## The issue
+
+We ran a diary study with participants who were early in the process of deciding whether training to become a teacher was right for them, and if so which courses to consider.
+
+When encountering the Find website, several participants made comments suggesting that the current design, where they are asked a series of questions before seeing matching courses, was off-putting or didn't work for them.
+
+Some of the reasons for this were:
+
+* they did not have a specific location in mind, as they were willing to move
+* the questions made them worry they were about to start an application, but were not yet ready to commit
+* they didn't yet understand how the process of finding and applying for a course worked
+* the GOV.UK branding looked 'official' (like submitting taxes) compared to the marketing websites
+
+## A hypothesis
+
+We developed a hypothesis that switching from using form-based components (radios, checkboxes and button) to using plain content, links and stand-alone buttons would give users more confidence to navigate between pages without feeling like they were making firm decisions.
+
+## What we changed
+
+We switched the questions about age group (primary or secondary), subject and location to use links and buttons instead of radio buttons or checkboxes.
+
+### First page
+
+We changed the first page of the service, so that instead of showing three radio button options, we had a lot more content and links on the page.
```

Comment:
```suggestion
We changed the first page of the service, so that instead of showing 3 radio button options, we had a lot more content and links on the page.
```